src/logic/Preferences.swift

```diff
@@ -193,6 +196,8 @@ class Preferences {
                             migrateMenubarIconFromCheckboxToDropdown()
                             // "Show minimized/hidden/fullscreen windows" checkboxes were replaced with dropdowns
                             migrateShowWindowsCheckboxToDropdown()
+                            // "Sort windows" checkboxes were replaced with dropdowns
+                            migrateSortWindowsCheckboxToDropdown()
```

Comment:
I'm a bit confused by this. Migration have been added in the past when we have saved a preference with the user already. Let's say we have a checkbox yes/no. Later, we change that checkbox to a dropdown menu with yes/no/maybe. We now need to **migrate** the users who already set is to yes or no, to the new serialization of yes/no from the dropdown.

In other words, migrations make sense *after* a feature has been introduced, when it evolves later. Here you're releasing the new preference, so no need to migrate anything.

Reply:
Yeah, I figured that, but given this was my first Swift and I wanted to see something work, I was just trying to get it all wired up.  I'd love to not do it this way, but am unsure ATM how to write one with no migration.  I don't recall seeing one like that.

Reply:
Figured it out...